src/displayapp/screens/SystemInfo.cpp

```diff
@@ -98,7 +95,7 @@ std::unique_ptr<Screen> SystemInfo::CreateScreen1() {
           dateTimeController.Day(), static_cast<uint8_t>(dateTimeController.Month()), dateTimeController.Year(),
           dateTimeController.Hours(), dateTimeController.Minutes(), dateTimeController.Seconds(),
           uptimeDays, uptimeHours, uptimeMinutes, uptimeSeconds,
-          batteryPercent, brightness, resetReason);
+          (int) batteryPercent, brightness, resetReason);
```

Comment:
The cast isn't needed here any more, as it is done as part of L46...

Reply:
Not a big deal, but I prefer the C++ syntax  :)
`auto batteryPercent = static_cast<uint8_t>(batteryController.PercentRemaining());`